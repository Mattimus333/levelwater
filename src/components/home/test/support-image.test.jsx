import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SupportImage from '../support-image.jsx';

describe('support image', () => {
  it('renders a snapshot', () => {
    const tree = shallow(<SupportImage />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
