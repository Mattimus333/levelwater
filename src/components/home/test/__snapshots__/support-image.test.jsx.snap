exports[`support image renders a snapshot 1`] = `
<div
  id="support-image" />
`;
